HMCL/src/main/resources/assets/lang/I18N_zh.properties

```
@@ -546,181 +546,180 @@ java.install=安裝 Java
 java.install.archive=源路徑
 java.install.failed.exists=該名稱已被使用
 java.install.failed.invalid=該檔案不是合法的 Java 安裝包，無法繼續安裝。
-java.install.failed.unsupported_platform=此 Java 與當前平臺不相容，無法安裝。
+java.install.failed.unsupported_platform=此 Java 與目前平臺不相容，無法安裝。
 java.install.name=名稱
-java.install.warning.invalid_character=名稱中包含非法字元
+java.install.warning.invalid_character=名稱中包含無效字元
 java.reveal=瀏覽 Java 目錄
 java.uninstall=移除此 Java
 java.uninstall.confirm=你確定要移除此 Java 嗎？此操作無法復原！
 
-lang=正體中文
+lang=繁體中文 (台灣)
```

```suggestion
lang=繁體中文
```

本文件可以以台湾地区用词为准，但并不是特供于台湾地区，我觉得不标出地区更好。